Citizen Web3.md

```diff
@@ -0,0 +1,42 @@
+## Validator Directory: Citizen Web3
+
+1) The name of your validator
+
+- Citizen Web3
+
+2) Why you want to become a validator for GovGen and AtomOne
+
+- We believe that decentralization is a superior form of communication stylistics rather than a centralized one. Our choice of the networks we validate (https://www.citizenweb3.com/staking) follows that goal. We want to become contributors to the networks that experiment with governance, build decentralized infrastructure, bridges between ecosystems. Both networks in our opinion fall into the first category heavily. Such a powerful network as Atom, is the building essence of a decentralized community. If blockchains are to be viewed as digital nations, atom is certainly a blue chip. We believe that by contributing to such a network, we help society to achieve the goal of decentralization. 
+
+3) Brief history/Intro to your validator
+
+- https://www.citizenweb3.com/about
+
+4) Details of your validator (website and contact email)
+
+- Website: https://www.citizenweb3.com/
+- Email: citizenweb3@protonmail.com
+
+5) Team structure, roles, and Discord handles
+
+- Citizen web3 is made of several parts. The infrastructure part of the project consists of 2 people. Myself, the founder and a devops. The podcast part has 2 more people. And the validator info project has 4 more people working on it. 3 developers and 1 designer. 
+Discord handles: serejandmyself_citizenweb3 | alibabaedge
+
+6) Link to your main social accounts
+
```

Comment:
7) Unique validator key to submit (from Cosmos Hub)

- cosmosvaloper1e859xaue4k2jzqw20cv6l7p3tmc378pc3k8g2u (from gaia)

(using different keys)

8) Region you and/or your validator is based (country)

- We are not bound to a single region. Mostly our operations are bound to Western Europe (Portugal)